datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/GmsGraphQLEngine.java

```diff
@@ -91,6 +96,21 @@
             DATA_JOB_TYPE
     );
 
+    /**
+     * Configures the graph objects that cannot be fetched by primary key
+     */
+    public static final List<LoadableType<?>> PROPERTY_TYPES = ImmutableList.of(
```

Comment:
Hmmm. Any other name than property types... To capture the idea that these are "expensive" fields of a graph entity, which require resolution via downstream services

Why do we differentiate here? Is EntityType a specific version of LoadableType?

Reply:
Yes, we need entity type as its own set to pass into the EntityTypeResolver. As for naming of the non-entity resolvable types, open to suggestions!